UI/EncyclopediaDetailPanel.cpp

```diff
@@ -53,7 +53,7 @@ namespace {
     const int DESCRIPTION_PADDING(3);
 
     void    AddOptions(OptionsDB& db)
-    { db.Add("UI.autogenerated-effects-descriptions", UserStringNop("OPTIONS_DB_AUTO_EFFECT_DESC"),  false,  Validator<bool>()); }
+    { db.Add("UI.dump-effects-descriptions", UserStringNop("OPTIONS_DB_DUMP_EFFECTS_GROUPS_DESC"),  false,  Validator<bool>()); }
```

Comment:
I think this string is missing, probably because I requested the original be used instead...